docs/maps/import-geospatial-data.asciidoc

```diff
@@ -58,6 +58,54 @@ On the {kib} home page, you can upload a file and import it into an {es} index w
 . Use the file chooser to select a GeoJSON file.
 . Click *Import file*.
 
+[discrete]
+=== Draw features in map
+
+Upload features into Elasticsearch by drawing lines, polygons, circles, bounding boxes, and points in a map.
+
+To create a new index for drawing:
+
+. <<maps-create, Create a new map>>.
+. Click *Add layer*.
+. Select *Create index*.
+. Set *Index name*.
+. Click *Create index* button.
+
+To open an existing index for drawing:
+
+. <<maps-create, Create a new map>>.
+
+. Click *Add layer*.
+
+. Select *Documents*.
+
+. Select *Data view* that points to your index. A <<data-views, data view>> can point to one or more indices. For feature editing, the data view must point to a single index.
+
+. Click *Add layer*.
+
+. Set *Scaling* to *Limit results to 10,000*.
+
+. In **Filtering**:
+** Clear the *Apply global search to layer data* checkbox.
+** If your data view contains a default time field, clear the *Apply global time to layer data* checkbox.
+
+. Click *Save & close*.
+
+. Click layer name in legend and select *Edit features*.
+
+When feature editing is open, a feature editing toolbox is displayed on the left side of the map.
```

Comment:
```suggestion
A feature editing toolbox appears on the left side of the map.
```

Reply:
Suggest indenting this sentence and the image under step 9.  To do this, add a "+" in the empty line.

Reply:
This sentence and image apply to both "To create a new index for drawing" and "To open an existing index for drawing" set of instructions. For this reason, I did not indent the sentence and image under step 9.